test/integration/targets/azure_rm_virtualmachine/main.yml

```diff
@@ -0,0 +1,21 @@
+- name: Setup for Azure VM tests
+  hosts: azure_testhost_1
```

Comment:
To make it more clear this isn't specific to any one sequence of tests I'd use implicit localhost here.